app/assets/stylesheets/application.sass

```diff
@@ -0,0 +1,18 @@
+h1
+    text-align: center
+    
+.grid-container
+    display: grid
+    grid-template-columns: 1fr 1fr
+    grid-gap: 30px
+    justify-content: space-around
+
+.grid-content
+    width: 350px
+    height: 350px
+    border: solid black
+    margin: auto
+
+    img 
```

Comment:
perhaps move this block above . grid-container (line 4) and then reduce indentation?

Reply:
re: image block from line 16 onwards

Reply:
Done